### DISCUSSION TITLE
Quality control

### DISCUSSION THREAD

**Discussion by shahesam84** [hopollo/OBS-Youtube-Player#2] [Ideas]
Thank you for this great work. Is there a way to control the output quality/resolution? I can changed the output height and depth but the quality is not that good after resizing. Thanks

**Comment by hopollo** [comment]
Hello, thanks for the kind words, i added your suggestion in the new commit, you can now use **&quality=** and **&forcequality=**

https://github.com/hopollo/OBS-Youtube-Player/commit/17ca630f8303cf2f62005b6e3e7939dd491465e3

Please refer to the README doc for further instructions, have fun
